src/main/java/picard/analysis/artifacts/Transition.java

```diff
@@ -25,8 +26,30 @@
         this.call = call;
     }
 
+    /**
+     * Gets a the enum representing the transition from a 'reference" got a 'call' base.
+     *
+     * <p>For example, a transtion from 'A' to 'T' would return {@link #AtoT}.
+     *
+     * @param ref reference base.
+     * @param call call base.
+     *
+     * @return enum representation for the transition-
+     */
     public static Transition transitionOf(final char ref, final char call) {
-        return transitionIndexMap[baseIndexMap[ref]][baseIndexMap[call]];
+        return transitionIndexMap[baseIndexOf(ref, () -> "ref")][baseIndexOf(call, () -> "call")];
```

Comment:
while this is nice, I would like to keep the efficiency of the original implementation (two fewer function calls). Could you simply wrap the original 
```
 transitionIndexMap[baseIndexMap[ref]][baseIndexMap[call]];
```
expression with a try-catch? I realize that the error message will suffer a little from that...but that's OK.

Reply:
Ok. Removed the function and changed for a try-catch with a more generic message.